copier.yaml

```diff
@@ -1,6 +1,10 @@
 _subdirectory: template_content
 _templates_suffix: ".jinja"
 
+_tasks:
+  - "python -c \"print('WARNING: This template has been deprecated. Beaker for Algorand Smart Contract development is no longer supported.\\nPlease use Algorand Python instead. The Algorand Python template can be found at https://github.com/algorandfoundation/algokit-python-template')\""
```

Comment:
This tasks runs before the init finishes but the files are still copied. Is there a way to run the tasks before prompting for the questions?